docs/examples/junit4/generic/src/test/java/generic/WaitStrategiesTest.java

```diff
@@ -14,53 +14,41 @@ public class WaitStrategiesTest {
 
     @Rule
     // waitForNetworkListening {
```

Comment:
Something is wrong with the docs for this one:
https://deploy-preview-5390--testcontainers.netlify.app/features/startup_and_waits/

(we should also check the docs pages for every other file that we touched, because we are using the comment-based code includes)